### PR TITLE
Update nunit 4.0 release notes

### PR DESCRIPTION
**Pull request by OsirisTerje**
Autogenerated release notes for nunit 4.0

### CODE REVIEW

**Comment by SeanKilleen** [comment]
@OsirisTerje the reason this failed the line length check is that I didn't realize it was automated and fixed it manually last time.

I'll add a commit that ignores the rule for this file.

File: docs/articles/nunit/toc.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@
   href: intro.md
 - name: NUnit 4 plans
   href: Towards-NUnit4.md
+- name: NUnit 3 to 4 migration guide
```
Comment (SeanKilleen) [inline docs/articles/nunit/toc.yml:5]:
```suggestion
- name: Migration Guides
```

This should probably be 1 page for all migration guides. We may want to backport some eventually or build upon them in 4.x -> 5.x.

File: docs/articles/nunit/release-notes/breaking-changes.md [hunk 2]
```diff
@@ -15,6 +15,9 @@ uid: breakingchanges
   * [FileAssert](../writing-tests/assertions/classic-assertions/File-Assert.md)
 * Assert.That overloads with _format_ specification and `params` have been removed in favor of an overload using
   `FormattableString`.
+* The lowest framework version supported is now version 4.6.2 and .Net 6.0.
+
+See full list of [deprecated issues](https://docs.nunit.org/articles/nunit/release-notes/framework.html#deprecated-features) and [breaking changes](https://docs.nunit.org/articles/nunit/release-notes/framework.html#the-following-issues-are-marked-as-breaking-changes) in the framework.
```
Comment (SeanKilleen) [inline docs/articles/nunit/release-notes/breaking-changes.md:20]:
```suggestion
See full list of [deprecated features](https://docs.nunit.org/articles/nunit/release-notes/framework.html#deprecated-features) and [breaking changes](https://docs.nunit.org/articles/nunit/release-notes/framework.html#the-following-issues-are-marked-as-breaking-changes) in the framework.
```

File: .devcontainer/devcontainer.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
 				"ms-dotnettools.csharp",
 				"github.vscode-github-actions",
 				"ms-azuretools.vscode-docker",
-				""
+				"stkb.rewrap"
```
Comment (SeanKilleen) [inline .devcontainer/devcontainer.json:21]: ℹ️ This ensures re-wrap is available in Codespaces as well as VS Code.

**Comment by SeanKilleen** [comment]
@OsirisTerje sorry about that, I merged the PR by accident because I had it open on my phone. It was fine as-is and I'll incorporate my tiny follow up notes as a separate PR.

**Comment by OsirisTerje** [comment]
I'm fine with that!  But awesome if you fix up my half-English :-)

**Comment by Siljaaa** [comment]
Hi :) 

I am not able to run tests in Visual Studio (.NET7) after upgrading NUnit to version 4.

**Comment by OsirisTerje** [comment]
@Siljaaa Please don't raise issues in a closed pull request in the docs repo.  This belongs in the nunit repo.

**Comment by Siljaaa** [comment]
@OsirisTerje I apologize, I am so sorry

@manfred-brands Thank you, yes this helped

**Comment by OsirisTerje** [comment]
@Siljaaa No need to apologize :-)  Just trying to point you to where you normally will get most help.  Good to hear you're unblocked!    Btw,  -adev.2 is out on myget. https://www.myget.org/feed/nunit/package/nuget/NUnit/4.1.0-adev.2

**Pull request merged**